addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -267,6 +282,9 @@ def write(self, vals):
         if 'stage_id' in vals:
             vals['date_last_stage_update'] = fields.Datetime.now()
             vals.update(self._onchange_stage_id_internal(vals.get('stage_id'))['value'])
+            # reset kanban state when changing stage
```

Comment:
I think the comment is not necessary as it is just explaining what the next two lines do.